### ISSUE TITLE
Do not open automatically on MacOs 10.14 

### ISSUE THREAD

**Issue by Octave-Laurentin** [eczarny/spectacle#941]
Even if you checked 'start at begining' as the screenshot below. You always need to find the application and double-click to activate it. 

![Capture d’écran 2020-03-12 à 16 09 14](https://user-images.githubusercontent.com/61840992/76536028-12104580-647c-11ea-8cc3-14e033225e91.png)

**Comment by xdvrx1** [comment]
yup, I think it's one bug, but the app still works, and it is fine for me
